tests/Integration/Database/Sqlite/DatabaseSqliteConnectionTest.php

```diff
@@ -10,9 +10,11 @@
 
 class DatabaseSqliteConnectionTest extends DatabaseTestCase
 {
-    protected function getEnvironmentSetUp($app)
+    protected function defineEnvironment($app)
     {
-        if (getenv('DB_CONNECTION') !== 'testing') {
+        parent::defineEnvironment($app);
+
+        if ($this->driver !== 'sqlite') {
```

Comment:
This changes allow the TestCase to be executed on persistent and in-memory SQLite

Reply:
Thanks!